system-tests/azure-blob-transfer-fixtures/src/testFixtures/java/org/eclipse/edc/test/system/blob/ConsumerConstants.java

```diff
@@ -28,4 +33,14 @@ public interface ConsumerConstants {
     int PROTOCOL_PORT = getFreePort();
     String PROTOCOL_PATH = "/protocol";
     String PROTOCOL_URL = "http://localhost:" + PROTOCOL_PORT + PROTOCOL_PATH;
+    Map<String, String> CONSUMER_PROPERTIES = Map.ofEntries(
```

Comment:
just a hint: `Map.of` is limited w.r.t. to the amount of entries that can be passed. In the spirit of future-proofing, i'd use a Map initializer:
```java
Map<String, String> myMap = new HashMap<String, String>() {{
    put("a", "b");
    put("c", "d");
}};
```

Reply:
Thanks for the hint - changed it.